Add Py3 Support

thealphadollar opened the pull request:
This PR addresses the following issues:

- fixes #32 : Added Pipfile and Pipfile.lock.
- fixes #10 : Used Robobrowser for the purpose of scraping.
- Added instructions in README regarding JSESSIONID.
- Other minor improvements.

**change severity: critical**
Tested on _Debian 9 Stretch_ with _Python 3.6.5_

thealphadollar committed:
Shift to pipenv standards

thealphadollar committed:
Add lxml as dependency for beautifulsoup

thealphadollar committed:
Add instructions to set JSESSIONID variable

thealphadollar committed:
Add RoboBrowser to support Py3

thealphadollar committed:
Test, refactor and rectify code

thealphadollar committed:
Update README with new pipenv instructions

thealphadollar commented:
cc @themousepotato

themousepotato commented:
@thealphadollar Thanks for the PR! Dependencies are not installing on testing this using the commands you mentioned in the README. Could you check that again?

thealphadollar committed:
Update README & improve error handling

thealphadollar commented:
@themousepotato Done :) 

- Added new instructions in README
- Added error check for missing "JSESSIONID" since os.getenv will return empty string but not give error if "JSESSIONID" is not present.

themousepotato commented:
Still not installing @thealphadollar :(

thealphadollar commented:
Robobrowser was not added to requirements.txt since it is not more the standard package handler for python.

themousepotato commented:
@thealphadollar Thanks for working on this! Merged and deployed :)
